pkg/credentials/credentials.go

```diff
@@ -9,7 +9,7 @@ import (
 	velerov1 "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
For some reason my extension now adds v1 on save when go extension for vscode is enabled.. and
`lint-fix doesn't work`

```
❯ make lint-fix 
/Users/tiger/oadp-operator/bin/golangci-lint run --fix
pkg/credentials/credentials.go:12:2: redundant-import-alias: Import alias "v1" is redundant (revive)
        v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
        ^
make: *** [lint-fix] Error 1
```

Reply:
make lint-fix doesn't fixes lint

Reply:
in this specific case can be a race condition with IDE, import related should be fixed automatically by `make lint-fix`

if it VSCode, I added this in my `setting.json` to avoid it changing things not following linter directives
```json
   "[go]": {
        "editor.codeActionsOnSave": {
            "source.organizeImports": "never"
        }
    },
```

Reply:
I saw this a few times too, I just renamed v1 to metav1 and it passed the lint.